webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const webpack = require('webpack');
+const path = require('path');
+
+module.exports = {
+  entry: {
+    'react-ga': './src/index.js',
+    'react-ga.min': './src/index.js'
+  },
+  output: {
+    path: path.resolve('./dist'),
+    filename: '[name].js'
+  },
+  module: {
+    loaders: [
+      { test: /\.js$/, loader: 'babel-loader', exclude: /node_modules/ },
```

Comment:
These two lines can be combined into one using `test: /\.jsx?$/`

Reply:
Thanks for catching that! I'll fix before the merge.

My npmjs username is "simeonc" https://www.npmjs.com/~simeonc, thanks.